lib/config/index.js

```diff
@@ -809,19 +809,28 @@ function initPrompts(newProject) {
       }
     })
     .then(function(transpiler) {
-      if (transpiler === curTranspiler || transpiler == 'none')
+      if (transpiler === 'none')
```

Comment:
If I remember correctly, skipping reconfiguration here is important in the case when the transpiler is already configured with manual / custom configuration.